libraries/ESP8266mDNS/ESP8266mDNS.cpp

```diff
@@ -261,6 +261,19 @@ bool MDNSResponder::addServiceTxt(char *name, char *proto, char *key, char *valu
   return false;
 }
 
+void MDNSResponder::clearServiceTxt(char *name, char *proto){
```

Comment:
why don't you use "const char*" here ?

Reply:
Good question. Mainly because I'm a newb and I was basing it off the structure of addServiceTxt.  I will change it to const char*